embark-ui/src/containers/FileExplorerRowContainer.js

```diff
@@ -64,7 +64,7 @@ class FileExplorerRowContainer extends React.Component {
 
   render() {
     return (
-      <div style={this.props.style.container}
+      <div style={this.props.style.container[0]}
```

Comment:
Just for clarification (as there's not description how this commit fixes the bug): why are we changing to `[0]` here?

Reply:
Because the library passes an array. Chrome handle that correctly, Firefox don't.